### ISSUE TITLE
setAutoRepeat false not working, keeps repeating

### ISSUE THREAD

**Issue by hannojg** [Aghajari/AXrLottie#20]
### Example reproduction code:
```java
setLottieDrawable(
  AXrLottieDrawable.fromJson(jsonString, cacheKey)
      .setSize(decodeWidth, decodeHeight)
      .setAutoRepeat(false)
      .setSpeed(speed) // speed is set to 1.0f
      .build()
);

playAnimation();
```

### Observation vs. Expectation

I expect the animation to only play once, and then stop. However, the animation keeps repeating.

**Comment by mojtabasheikhy** [comment]
I have the same problem

**Comment by Aghajari** [comment]
Hi @mojtabasheikhy and @hannojg,
Auto repeat is based on `AXrLottieImageView`'s autoRepeat property.
`AXrLottieDrawable`'s autoRepeat only matters if you are not using `AXrLottieImageView`.
Or you can change the auto repeat on `AXrLottieDrawable` after calling  `AXrLottieImageView#setLottieDrawable`.

This code should work: 

```java
lottieView.setAutoRepeat(false);

lottieView.setLottieDrawable(
  AXrLottieDrawable.fromJson(jsonString, cacheKey)
      .setSize(decodeWidth, decodeHeight)
      .build()
);

lottieView.playAnimation();
```

let me know if it didn't. Thanks!

**Comment by Aghajari** [comment]
Fixed on v1.4.0

**Comment by mojtabasheikhy** [comment]
> Hi @mojtabasheikhy and @hannojg, Auto repeat is based on `AXrLottieImageView`'s autoRepeat property. `AXrLottieDrawable`'s autoRepeat only matters if you are not using `AXrLottieImageView`. Or you can change the auto repeat on `AXrLottieDrawable` after calling `AXrLottieImageView#setLottieDrawable`.
> 
> This code should work:
> 
> ```java
> lottieView.setAutoRepeat(false);
> 
> lottieView.setLottieDrawable(
>   AXrLottieDrawable.fromJson(jsonString, cacheKey)
>       .setSize(decodeWidth, decodeHeight)
>       .build()
> );
> 
> lottieView.playAnimation();
> ```
> 
> let me know if it didn't. Thanks!

Thanks, it's fixed 🙌
